browser/permissions/brave_ethereum_permission_context_browsertest.cc

```diff
@@ -74,7 +74,7 @@ class BraveEthereumPermissionContextBrowserTest : public InProcessBrowserTest {
   void SetUpOnMainThread() override {
     mock_cert_verifier_.mock_cert_verifier()->set_default_result(net::OK);
     host_resolver()->AddRule("*", "127.0.0.1");
-    https_server()->ServeFilesFromDirectory(GetChromeTestDataDir());
```

Comment:
`ServeFilesFromSourceDirectory` converts arg to absolute path, so test binary could be run from everywhere.